js/script.js

```diff
@@ -0,0 +1,89 @@
+import {
+  $taskForm,
+  $taskTitle,
+  $taskList,
+  $btnCreate,
+} from "../js/elements.js";
+import { stripSanitizedParts } from "../js/utils/stripSanitizedParts.js";
+import { formatDate } from "../js/helpers/formatDate.js"; // Import the formatDate function
+
+let tasks = [];
+
+const openForm = () => {
+  $taskForm.classList.add("show");
+};
+
+const closeForm = () => {
+  $taskForm.classList.remove("show");
+};
+
+const addTask = () => {
+  const title = $taskTitle.value.trim();
+
+  if (title === "") {
+    showError("Please enter a task title.");
+    return;
+  }
+
+  const taskTitle = stripSanitizedParts(title);
+
+  const newTask = {
+    id: Date.now(),
+    title: taskTitle,
+    createdAt: formatDate(new Date()), // Use the imported formatDate function
```

Comment:
Ditto

Reply:
For reducing the possibility of confusion.